cv_bridge/python/cv_bridge/core.py

```diff
@@ -169,12 +169,13 @@ def imgmsg_to_cv2(self, img_msg, desired_encoding='passthrough'):
         dtype, n_channels = self.encoding_to_dtype_with_channels(img_msg.encoding)
         dtype = np.dtype(dtype)
         dtype = dtype.newbyteorder('>' if img_msg.is_bigendian else '<')
+        buf = np.asarray(img_msg.data, dtype=dtype)
```

Comment:
It's better add a condition checking to `img_msg.data` before the real conversion to a byte-like object, thanks !